src/util/mod.rs

```diff
@@ -44,6 +45,25 @@ impl DataWriter {
         for point in points { write!(file, "{},{}\n", point.x, point.y)?; }
         Ok(())
     }
+
+    /// Creates a new file in the writers directory with each body written on a separate line.
+    pub fn write_body_file(&mut self, bodies: Vec<Body>) {
+        let path = format!("{}/frame-{}.txt", self.directory, self.counter);
```

Comment:
The idea to write the body data into a file is to be able to save the initial state of the simulation. That means we only need to create one single file. I think you're confusing the need to write frames with point data with the need to save the list of bodies.